src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  position: relative;
+}
+
+.nav__list {
+  text-decoration: none;
+  margin-left: 20px;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  padding: 23px 0;
+  line-height: 60px;
+}
+
+.nav__list:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.nav__ul {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+}
+
+.is-active {
+  position: relative;
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active::after {
+  content: " ";
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: rgba(0, 172, 220, 1);
+  bottom: 0;
+  position: absolute;
+  display: block;
+  left: 0;
+}
+
+.logo {
+  margin: 10px 0;
+}
```

Comment:
I don't think that you really need this margins since you've aligned elements
Also `display: flex` will fix additional bottom space of the link